bears/general/FilenameBear.py

```diff
@@ -74,12 +82,38 @@ def run(self, filename, file,
         if ignore_uppercase_filenames and filename_without_extension.isupper():
             return
 
-        if new_name != filename_without_extension:
-            diff = Diff(file, rename=os.path.join(head, new_name + extension))
+        messages = []
 
-            yield Result(
-                self,
+        if new_name != filename_without_extension:
+            messages.append(
                 'Filename does not follow {} naming-convention.'.format(
-                    file_naming_convention),
-                diff.affected_code(filename),
-                diffs={filename: diff})
+                    file_naming_convention))
+
+        if not filename_without_extension.startswith(filename_prefix):
+            new_name = filename_prefix + new_name
+            messages.append(
+                'Filename does not use the {} prefix.'.format(
+                    filename_prefix))
+
+        if not filename_without_extension.endswith(filename_suffix):
+            new_name = new_name + filename_suffix
+            messages.append(
+                'Filename does not use the {} suffix.'.format(
```

Comment:
maybe displaying the `repr`? -->
`Filename does not use the {!r} suffix.`

What do you think?